openff/toolkit/topology/molecule.py

```diff
@@ -275,7 +275,7 @@ def add_bond(self, bond):
 
         Parameters
         ----------
-        bond: an openff.toolkit.topology.molecule.Bond
+        bond: an penff.toolkit.topology.molecule.Bond
```

Comment:
Typo?

Reply:
Yep, typing too quickly while moving around in files